### DISCUSSION TITLE
Order Delivery feature

### DISCUSSION THREAD

**Discussion by minhcasi** [inspireui/support#5054] [Working on Features]
Integrate Fluxstore with https://woocommerce.com/products/woocommerce-order-delivery/

**Comment by HarrySmith01** [comment]
This one looks similar to https://github.com/inspireui/support/discussions/5052

But this one is very expensive and with almost no features.
If someone ask me to value this plugin then I would not even pay $5 for this plugin. 

It does not do any thing expect adding delivery date or time option on Checkout page but the plugin  link I shared on https://github.com/inspireui/support/discussions/5052 actually provides more control on time slot. 

With those two plugins, you can decide when or not  a delivery slot should appear. you can control processing time. You can restrict delivery slot based on processing time and some other logic.

For example: If I need 4 hour to process order then I can set a delivery slot will be viable after the processing time so if Current time is 4 PM then users will be able to place order for 8 PM delivery slot. and they wont be able to see any delivery slot before 8 PM,

**Comment by Haniel5** [comment]
Woocommerce app supports https://wordpress.org/plugins/order-delivery-date-for-woocommerce/
FluxStore Shopify app http://1.envato.market/o9vAg also supports this feature 👍

**Reply by shubhamrane48** [reply]
Hi,
Woocommerce app supports https://wordpress.org/plugins/order-delivery-date-for-woocommerce/
In above plugin this is only supported on delivery date in fluxstore.
for time slot this plugin is supported in website but in fluxstore application it is not showing the slots.
